omnigibson/prims/rigid_prim.py

```diff
@@ -162,8 +162,9 @@ def update_meshes(self):
             if prim.GetPrimTypeInfo().GetTypeName() in GEOM_TYPES:
                 mesh_name, mesh_path = prim.GetName(), prim.GetPrimPath().__str__()
                 mesh_prim = get_prim_at_path(prim_path=mesh_path)
-                mesh_kwargs = {"prim_path": mesh_path, "name": f"{self._name}:{mesh_name}"}
-                if mesh_prim.HasAPI(UsdPhysics.CollisionAPI):
+                is_collision = mesh_prim.HasAPI(UsdPhysics.CollisionAPI)
+                mesh_kwargs = {"prim_path": mesh_path, "name": f"{self._name}:{'collision' if is_collision else 'visual'}_{mesh_name}"}
```

Comment:
hmm is this a bug that we just haven't caught?

Reply:
Yeah -- usually there's no naming collision from how we programmatically generate our USDs, but for some custom USDs I used, there ended up being naming collisions since the visual and collision geoms shared the same name